utils/build-script-impl

```diff
@@ -1491,19 +1516,26 @@ function set_swiftpm_bootstrap_command() {
 
 
 for host in "${ALL_HOSTS[@]}"; do
+    # Skip this pass when the only action to execute can't match.
+    if [[ "${ONLY_EXECUTE}" != "" ]] && ! [[ "${ONLY_EXECUTE}" == ${host}-*-build ]]; then
```

Comment:
Do we need to compare against the empty string ("all" in my proposal)?  If the string is empty, it can't match the pattern either.